docs/usage/CompatibleOCPVersions.md

```diff
@@ -6,7 +6,7 @@ Refer to the guidelines below to select the correct mig-operator version for you
 
  - **OpenShift 3.7 - 4.5**
    - Crane 1.5.x is compatible
-   - Install Crane using [mig-legacy-operator](https://github.com/konveyor/mig-legacy-operator) available from YAML manifests.
+   - Install Crane using [mig-operator 1.5.x](https://github.com/konveyor/mig-operator/tree/release-1.5.1) available from YAML manifests.
```

Comment:
While the link goes to release-1.5.1 branch, is it clear enough to simply state 1.5.x here?  Should we more clearly indicate this is not available in 1.5.0 -- or first available in 1.5.1?

Reply:
I'm lost. For 3.7-4.5 we should be telling people to use mig-legacy-operator, correct? appregistry is frozen. We can't do any new releases for 4.1-4.5 via OLM/appregistry.

Reply:
That is correct.  Because it's been frozen we no longer backport into appregistry and MTC/Crane 1.5.1+ using mig-legacy-operator (non-OLM) `operator{,-3.7}.yml` & `controller.yml` is the path for MTC on <=OCPv4.5 clusters.  MTC/Crane 1.6 (1.6.0) will be made available via OLM on v4.6-v4.9 clusters.

Reply:
I thought the consensus on the call the other day was that mig-legacy-operator is not yet productized and in ready state to be released as the 3.7 - 4.5 solution in the MTC 1.6 release timeframe. 

Transposed Eriks notes from that call here
https://issues.redhat.com/browse/MIG-783